packages/oas/medusa-oas-cli/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@medusajs/medusa-oas-cli",
+  "version": "0.1.0",
+  "description": "OAS CLI",
+  "main": "dist/index.js",
+  "bin": {
```

Comment:
todo: `"files": ["dist"]`

Reply:
Done in b05953ed70696f10cb783522fde39362906578d6